### PR TITLE
Support for ADVA FSP150 EGX and GE11x devices - Temp Sensors

### PR DESCRIPTION
**Pull request by czilian**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by laf** [comment]
That's a lot of mibs!

I can already see duplicated mibs like LLDP-MIB.mib so you'll need to remove all of the ones we have as standard already.

**Comment by czilian** [comment]
Got it. Will do. Thx,

File: includes/definitions/advafsp150eg-x.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+os: advafsp150eg-x
```
Comment (laf) [inline includes/definitions/advafsp150eg-x.yaml]: This os name is too specific, especially seeing as you say it covers another device.

The OS should be named after the operating system / firmware which is run on these devices. If they don't have one then it should be the group of products this covers.

File: includes/definitions/advafsp150eg-x.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+text: ADVA FSP150 EGX
+type: network
+icon: adva
+group: MetroE
```
Comment (laf) [inline includes/definitions/advafsp150eg-x.yaml]: What's this? The group would be to tie in multiple OS' to one vendor, like cisco with ios, ios-xe, etc.

File: includes/definitions/advafsp150eg-x.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+over:
+    - { graph: device_bits, text: Traffic }
+    - { graph: device_temperature, text: Temperature }
+discovery:
+    - sysObjectId:
+      - .1.3.6.1.4.1.2544.1.12.1.1.7
+mib_dir:
+    - advafsp150eg-x
```
Comment (laf) [inline includes/definitions/advafsp150eg-x.yaml]: The mib_dir should be the manufacturers name ideally.

File: includes/definitions/advafsp150eg-x.yaml (hunk 1 above)
Comment (czilian) [inline includes/definitions/advafsp150eg-x.yaml]: Ok. Making a product just work or having it structured vendor-centric to follow the overall concept its what missing on my side. I will try to learn from your approach to APC. There you cover also quite a wide range of products with different readings. For now just bin my pull request. Hope you can help here and there by looking at my fork upon request before I place a pull request again. Thx

**Pull request closed without merging**